runtimes/runtimes/standalone.ts

```diff
@@ -184,7 +184,11 @@ export const standalone = (props: RuntimeProps) => {
                     },
                     error => {
                         console.error(error)
-                        process.exit(10)
+                        // arbitrary 5 second timeout to ensure console.error flushes before process exit
```

Comment:
Wouldn't you want to force-flush the console output and exit immediately? Waiting 5 more seconds seems like a side effect, one where a client could start sending what it intended to during the 2 second window of time the system was listening for data, only to have the process terminate.
```
process.stdout.write()
process.stderr.write()
```

Reply:
`process.stderr.flush()`?